tests/xlsx.test.js

```diff
@@ -0,0 +1,82 @@
+const xlsx = require('xlsx');
+
+function getMissingColumnHeaders(document) {
+    const wb = xlsx.readFile(__dirname + '/../util/' + document, {
+        sheetRows: 1,
+    });
+    const headersArray = xlsx.utils.sheet_to_json(wb.Sheets[wb.SheetNames[0]], {
+        header: 1,
+    })[0];
+    const headersSet = new Set(headersArray);
+
+    const requiredHeaders = [
+        'CODE',
+        'TITLE',
+        'TYPE',
+        'VENUE',
+        'FACULTY',
+        'CREDITS',
+        'SLOT',
+    ];
+    const missingHeaders = [];
+
+    for (let i = 0; i < requiredHeaders.length; ++i) {
+        if (!headersSet.has(requiredHeaders[i])) {
+            missingHeaders.push(requiredHeaders[i]);
+        }
+    }
+
+    return missingHeaders;
+}
+
+describe('does xlsx have the right headers', () => {
+    test('vellore', () => {
```

Comment:
This should simply skip the test. 
```suggestion
    test.skip('vellore', () => {
```

Reply:
That would skip it permanently, let it fail for now but in future PRs we'll try to make sure not to merge until it passes.

Basically what we want in future PRs is for the course list slots to be present in the timetable.

Reply:
We can't mark it as a required check if it's failing. 
No point of running tests when we know it's going to fail. We can always re-enable the test when we know it's working.